src/shogun/labels/BinaryLabels.cpp

```diff
@@ -64,6 +65,22 @@ CBinaryLabels::CBinaryLabels(CFile * loader) : CDenseLabels(loader)
 {
 }
 
+CBinaryLabels::operator CMulticlassLabels()
+{
+	CMulticlassLabels src;
+	
+	for (index_t i = 0; i < m_labels.vlen; i++)
+	{
+		if(m_labels.vector[i] == -1)
```

Comment:
Actually, sorry, it is in most cases.
But this needs checks and user readable error messages